core/controllers/suggestion.py

```diff
@@ -84,14 +89,41 @@ class SuggestionHandler(base.BaseHandler):
     @acl_decorators.can_suggest_changes
     def post(self):
         try:
-            suggestion_services.create_suggestion(
+            suggestion = suggestion_services.create_suggestion(
                 self.payload.get('suggestion_type'),
                 self.payload.get('target_type'), self.payload.get('target_id'),
                 self.payload.get('target_version_at_submission'),
                 self.user_id, self.payload.get('change'),
                 self.payload.get('description'))
         except utils.ValidationError as e:
             raise self.InvalidInputException(e)
+
+        html_list = suggestion.get_all_html_content_strings()
+        filenames = (
+            html_cleaner.get_image_filenames_from_html_strings(html_list))
+        for filename in filenames:
+            image = self.request.get(filename)
+            if not image:
+                logging.error(
+                    'Image not provided for file with name %s when the '
+                    ' suggestion with id %s was created.' % (
+                        filename, suggestion.suggestion_id))
+                raise self.InvalidInputException(
+                    'No image data provided for file with name %s.'
+                    % (filename))
+            try:
+                file_format = (
+                    image_validation_services.validate_image_and_filename(
+                        image, filename))
+            except utils.ValidationError as e:
+                e = '%s' % (e)
+                raise self.InvalidInputException(e)
+            image_is_compressible = (
+                file_format in feconf.COMPRESSIBLE_IMAGE_FORMATS)
+            fs_services.save_original_and_compressed_versions_of_image(
+                filename, feconf.ENTITY_TYPE_SUGGESTION,
+                suggestion.target_id, image, 'image', image_is_compressible)
```

Comment:
For reviewers:

We are expecting to store  images for suggestion in the following URL: `<bucket-name>/suggestion/<enitity-id>/assets/image>/<filname>`.

For more info check this doc: https://docs.google.com/document/d/1kGvPjYnW9ArYKtzxYOmFbp7MA-A45B3znRWFIlaEGVo/edit?usp=sharing